src/cdk/testing/tests/testbed.spec.ts

```diff
@@ -102,6 +102,13 @@ describe('TestbedHarnessEnvironment', () => {
           );
           expect(await (await harness.deepShadow()).text()).toBe('Shadow 2');
         });
+
+        it('should be able to retrieve the native DOM element from a UnitTestElement', async () => {
+          const harness = await TestbedHarnessEnvironment
+            .harnessForFixture(fixture, MainComponentHarness);
+          const element = TestbedHarnessEnvironment.getNativeElement(await harness.host());
+          expect(element.id).toContain('root');
```

Comment:
why not compare the tagName like you did in the protractor test?

Reply:
The html of `element` here looks like this, so the root element doesn't have the `test-main` tag like for the protractor test. Tbh, I have no clue why this happens

```html
<div id="root8" ng-version="10.1.0">
   <div class="click-test" style="width: 100px; height: 100px; background: grey;"></div>
   <div class="click-test-result">0-0</div>
    ...
</div>
```

Reply:
For contrast, this is what the html of the protractor element looks like

```html
<test-main>
   <div class="click-test" style="width: 100px; height: 100px; background: grey;"></div>
   <div class="click-test-result">0-0</div>
    ...
</test-main>
```

Reply:
ah ok, makes sense

Reply:
@mmalerba ಠಿ_ಠ